examples/PaginationExample.js

```diff
@@ -4,29 +4,24 @@
 
 "use strict";
 
-var FakeObjectDataListStore = require('./helpers/FakeObjectDataListStore');
-var FixedDataTable = require('fixed-data-table-2');
-var React = require('react');
-
-const {Table, Column, Cell} = FixedDataTable;
-
-const TextCell = ({rowIndex, data, col, ...props}) => {
-  var rowObject = data.getObjectAt(rowIndex);
-  return (
-    <Cell {...props}>
-      {rowObject ? rowObject[col] : 'pending'}
-    </Cell>
-  );
-};
+const FakeObjectDataListStore = require('./helpers/FakeObjectDataListStore');
+const { PagedCell } = require('./helpers/cells');
+const { Table, Column, Cell } = require('fixed-data-table-2');
+const React = require('react');
 
 class PagedData {
   constructor(callback) {
     this._dataList = new FakeObjectDataListStore(2000);
     this._end = 50;
     this._pending = false;
+    this._dataVersion = 0;
     this._callback = callback;
   }
 
+  getDataVersion() {
```

Comment:
Introduced a version field similar to what we do on LiveDesign so I could make the renderers pure for this example and get better performance